packages/circuits/README.md

```diff
@@ -45,7 +45,27 @@ This template provides functionality for performing arithmetic operations in fin
 
 ### utils.circom
 
-This template includes a collection of utility functions used across multiple circuits, such as bit manipulation functions, comparison functions, or conversion functions.
+The `utils.circom` file includes a collection of utility templates and functions that are used across multiple circuits. These utilities cover a wide range of functionalities, including bit manipulation, comparison, conversion, and arithmetic operations in finite fields. It serves as a foundational component for building complex arithmetic circuits.
+
+## Utility templates
+
+### bytes2ints.circom
+
+This template converts an array of bytes into an array of integers. It is designed to handle inputs of any byte size and outputs integers based on the number of bytes specified. This is particularly useful for processing large binary data within arithmetic circuits.
```

Comment:
We can mention that 31 bytes is converted as one integer - because the circom field max value is a ~31 byte number
Also the representation is in little endian order